src/main/java/org/jitsi/videobridge/SctpConnection.java

```diff
@@ -153,7 +153,8 @@ private static synchronized int generateDebugId()
     /**
      * Data channels mapped by SCTP stream identified(sid).
      */
-    private final Map<Integer,WebRtcDataStream> channels = new HashMap<>();
+    private final Map<Integer,WebRtcDataStream> channels
+        = new ConcurrentHashMap<>();
```

Comment:
No, removing the commit (or rather, not including it in the new version of the PR).